client/src/components/navigations/MainStackNavigator/ProfileModal.tsx

```diff
@@ -297,78 +298,101 @@ const ModalContent: FC<ModalContentProps> = ({modalState, hideModal}) => {
             right: 8,
             flexDirection: 'row',
           }}>
-          <TouchableOpacity
-            testID="touch-done"
-            onPress={() => {
-              navigation.navigate('MainStack', {
-                screen: 'Report',
-                params: {
-                  name: nickname || name || getString('NO_NAME'),
-                  userId: id,
-                },
-              });
-
-              hideModal();
-            }}>
-            <View
-              style={{
-                paddingRight: 12,
-                paddingLeft: 8,
-                paddingVertical: 8,
-              }}>
-              <FontAwesome name="exclamation-circle" size={24} color="white" />
-            </View>
-          </TouchableOpacity>
-          {isCreateBlockedUserInFlight || isDeleteBlockedUserInFlight ? (
-            <View
-              style={{
-                paddingRight: 16,
-                paddingLeft: 8,
-                paddingVertical: 8,
-                justifyContent: 'center',
+          {modalState.isMyself ? (
+            <TouchableOpacity
+              testID="profile-update-button"
+              style={{padding: 8}}
```

Comment:
Can we make this into a styled component?

Reply:
I'm afraid that I can't fully understand your intention
```
const StyledProfileUpdateButton = styled.TouchableOpacity`
  padding: 8px 16px 8px 8px;
`;
```
I can update the code as above, but I don't personally think that this is more readable or has any benefit.